src/main/java/com/fundynamic/d2tm/Game.java

```diff
@@ -35,6 +35,8 @@ public void initStatesList(GameContainer container) throws SlickException {
                 TILE_HEIGHT
         );
 
+        container.setVSync(true);
```

Comment:
Ok, let's keep it then. And buy a better Macbook, because I have no problems here :smiling_imp: